Packs/CommonWidgets/Scripts/GetLargestInvestigations/GetLargestInvestigations.yml

```diff
@@ -1,16 +1,33 @@
 args:
 - default: false
-  description: The end date for fetching incidents in ISO format. Incidents will be fetched until the last day of the specified month. For example, if you specify 2020-08-14, incidents created on the specified "from" date until  2020-08-31 will be returned.
+  description: The end date for fetching incidents in ISO format. Incidents will be
+    fetched until the last day of the specified month. For example, if you specify
+    2020-08-14, incidents created on the specified "from" date until  2020-08-31 will
+    be returned.
   isArray: false
   name: to
   required: false
   secret: false
 - default: false
-  description: The start date for fetching incidents in ISO format. Incidents will be fetched starting from the first day of the specified month. For example, if you specify 2020-08-14, incidents created on 2020-08-01 will be returned.
+  description: The start date for fetching incidents in ISO format. Incidents will
+    be fetched starting from the first day of the specified month. For example, if
+    you specify 2020-08-14, incidents created on 2020-08-01 will be returned.
   isArray: false
   name: from
   required: false
   secret: false
+- auto: PREDEFINED
+  default: false
+  defaultValue: 'false'
+  description: Change to true to return a result suitable for a table widget. By default
+    the return will be in Markdown.
```

Comment:
If "true" results are suitable for a table widget. If "false", results are returned in Markdown. Default is "false".